src/LandingPage.js

```diff
@@ -1,11 +1,37 @@
-import React, { Component } from 'react';
+import React from 'react';
 import Container from 'react-bootstrap/lib/Container';
+import './mainSection.css';
```

Comment:
Let's rename this file to match the component. Typically try to have stylesheets map 1:1.